backend/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM node:10-alpine
+
+# Set environment variables.
+ENV NODE_ENV production
+ENV DEBUG app:info,app:warn,app:error
+
+# Install dependencies.
+RUN apk update && \
+  apk add --no-cache \
+    --virtual build-dependencies \
+      build-base \
+      gcc \
+      wget \
+      git
+
+# Create unprivileged account.
+RUN adduser -D project
+
+# Switch working directory.
+WORKDIR /home/project
+
+# Copy source code.
+COPY app/ ./app
+COPY package.json yarn.lock* ./
+COPY config.js knexfile.js .babelrc ./
```

Seems like these might just as well be in the same layer.

```suggestion
COPY package.json yarn.lock* config.js knexfile.js .babelrc ./
```

 Also, why `yarn.lock*` and not just plain `yarn.lock`?

```suggestion
COPY package.json yarn.lock config.js knexfile.js .babelrc ./
```

---

Good point! I'll merge them.

The asterisk is so that a projekt doesn't need to have a lock-file in order to build. This is to be as forgiving as possible but that might be wrong :smiley: I'll remove it. Thank you!

---

Done